src/Select/Select.tsx

```diff
@@ -1,115 +1,108 @@
-import React, { useState, cloneElement, ReactElement, useEffect, Ref } from 'react'
+import React, { useState, useEffect } from 'react'
 import styled from 'styled-components'
 
 import { BlockProps } from '../Block'
 import { Dropdown, DropdownPlacement } from '../Dropdown'
-import { InputProps } from '../Input/Input'
-import { SelectInput } from '../SelectInput'
+import { SelectInput, SelectInputProps } from '../SelectInput'
 
 import { SelectListProps, SelectListData, SelectList } from '../SelectList'
```

Comment:
Since we're touching a lot of components already here, maybe we want to rename `SelectList` to `Options` to be more in line with the HTML native elements and the `OptionsComponent` prop?

cc @Michiel87 @Stefanwullems

Reply:
Alternatively, what about namespacing the Select component and include the default implementations as `Select.Input` and `Select.List` / `Select.Options` ?